x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/slm/SnapshotRetentionConfigurationTests.java

```diff
@@ -313,7 +313,7 @@ private SnapshotInfo makeInfo(long startTime) {
         meta.put(SnapshotsService.POLICY_ID_METADATA_FIELD, REPO);
         final int totalShards = between(1, 20);
         SnapshotInfo snapInfo = new SnapshotInfo(
-            new Snapshot(REPO, new SnapshotId("snap-" + randomAlphaOfLength(3), "uuid")),
+            new Snapshot(REPO, new SnapshotId("snap-" + randomUUID(), "uuid")),
```

Comment:
`randomAlphaOfLength(3)` was introduced in https://github.com/elastic/elasticsearch/pull/46407
I am surprised we have never seen this before for such a long time